bluebrain/repo-bluebrain/packages/py-diameter-synthesis/package.py

```diff
@@ -22,10 +23,15 @@ class PyDiameterSynthesis(PythonPackage):
     depends_on("py-setuptools-scm", type="build")
 
     depends_on("py-click@7.0:", type=("build", "run"))
-    depends_on("py-numpy@1.15.0:", type=("build", "run"))
-    depends_on("py-scipy@0.13.3:", type=("build", "run"))
-    depends_on("py-matplotlib@2.2:", type=("build", "run"))
-    depends_on("py-pandas@0.24:", type=("build", "run"))
+    depends_on("py-numpy@1.33.0:", type=("build", "run"), when="@0.5.3:")
+    depends_on("py-numpy@1.15.0:", type=("build", "run"), when="@:0.5.3")
```

Comment:
Ranges in Spack are _inclusive_, so for 0.5.3 either Numpy works. One of these lines should either read `@0.5.4:` or `@:0.5.2`

Reply:
Aaaaaah yes, I forgot again about that :facepalm: 
Thanks!!! :heart:

Reply:
Btw, can you remind me if it is possible to check the dependencies locally? Because the CI is super long... I feel like I already did this before but I can't remember.

Reply:
Do you mean the `check_python_module` thing?

I _think_ if you install the package locally in a Spack environment, add `py-pip`, you should be able to run `pip check` locally.